index.html

```diff
@@ -26,6 +26,14 @@ <h1 id="logo">Aziz Mejri</h1>
         </ul>
       </nav>
     </header>
+    <div id="mobile-menu" class="mobile-menu">
+      <button id="x-button" type="button">✕</button>
+      <ul>
+        <li><a class="nav-link-mobile" href="#projects">Portoflio</a></li>
+        <li><a class="nav-link-mobile" href="#about-me">About</a></li>
+        <li><a class="nav-link-mobile" href="#contackt">Contact</a></li>
```

Comment:
What you did looks good 👍🏽 but you must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Kindly make yours match the Figma please.

Your design:
![Screenshot 2023-01-05 at 08 03 29](https://user-images.githubusercontent.com/45739963/210786428-d35cdab5-3ab3-47c5-b1b3-a7c14dfc13fb.png)

The Figma
![Screenshot 2023-01-05 at 08 03 52](https://user-images.githubusercontent.com/45739963/210786501-cd8d7b93-4955-483d-a55e-3069093ed644.png)

I commented on this element because it seems as if the class selector applied here is not used anywhere in your CSS or JS. Make sure you use it to make your design match the Figma. 😉